client/my-sites/upgrade-nudge/docs/example.jsx

```diff
@@ -8,30 +8,29 @@ import React from 'react';
  */
 import UpgradeNudge from 'my-sites/upgrade-nudge';
 
-export default React.createClass( {
-
-	displayName: 'UpgradeNudge',
-
-	render: function() {
-		return (
+const UpgradeNudgeExample = () => {
+	return (
+		<div>
+			<div>
+				<UpgradeNudge
+					feature="custom-domain"
+					href="#"
+					shouldDisplay={ () => true }
```

Comment:
lodash has (of course 😉) a method for that: [`stubTrue`](https://lodash.com/docs#stubTrue). If we use it, that will mean we won't be creating new functions on every render and might help with performance (I know, this is devdocs, but still).